grpc/proto/v1/search_get.proto

```diff
@@ -227,6 +228,11 @@ message NearObject {
   optional double distance = 3;
 }
 
+message Rerank {
+  string property = 1;
```

Comment:
can this be a reference?

Reply:
We could make it optional as with the `query` property. Is this so that its type matches Weaviate's internal type?

From my understanding, the `property` is required by the reranking logic so it shouldn't be optional. Is this actually not the case?

Reply:
Ahhh, gotcha! That's a good question, I will test it out and see. What you're asking isn't documented but it could be possible

Reply:
I would guess that it doesn't work but better be sure :)

Reply:
[This loop](https://github.com/weaviate/weaviate/blob/master/usecases/modulecomponents/additional/rank/rank_result.go#L45) chooses the relevant properties from the schema and only considers the object itself. So no, it won't work with reference properties!